test_pytest.py

```diff
@@ -4,71 +4,116 @@
+#     Check that it returns the new education in that list
+#     '''
+#     example_education = {
+#         "course": "Engineering",
+#         "school": "NYU",
+#         "start_date": "October 2022",
+#         "end_date": "August 2024",
+#         "grade": "86%",
+#         "logo": "example-logo.png"
+#     }
+#     item_id = app.test_client().post('/resume/education',
+#                                      json=example_education)
 
-    response = app.test_client().get('/resume/education')
-    assert response.json[item_id] == example_education
+#     response = app.test_client().get('/resume/education')
+#     assert response.json[item_id] == example_education
 
 
-def test_skill():
-    '''
-    Add a new skill and then get all skills. 
+# def test_skill():
+#     '''
+#     Add a new skill and then get all skills. 
     
-    Check that it returns the new skill in that list
-    '''
-    example_skill = {
-        "name": "JavaScript",
-        "proficiency": "2-4 years",
+#     Check that it returns the new skill in that list
+#     '''
+#     example_skill = {
+#         "name": "JavaScript",
+#         "proficiency": "2-4 years",
+#         "logo": "example-logo.png"
+#     }
+
+#     item_id = app.test_client().post('/resume/skill',
+#                                      json=example_skill)
+
+#     response = app.test_client().get('/resume/skill')
+#     assert response.json[item_id] == example_skill
+
+
+'''
+Makes request to add a new experience and checks if the received payload have all required fileds with valid formats
+'''
+def test_add_experience():
+    response = app.test_client().post('/resume/experience', json={
+        "title": "Software Developer",
+        "company": "A Cool Company",
+        "start_date": "June 2022",
+        "end_date": "October 2024",
+        "description": "Writing Python Code",
+        "logo": "example-logo.png"
+    })
+    print(response.json)
+    assert response.status_code == 201
+    
+    
+def test_add_education():
+    response = app.test_client().post('/resume/education', json={
+        "course": "Computer Science",
+        "school": "University of Tech",
+        "start_date": "September 2019",
+        "end_date": "July 2022",
+        "grade": "A+",
         "logo": "example-logo.png"
-    }
+    })
+    print(response.json)
+    assert response.status_code == 201
 
-    item_id = app.test_client().post('/resume/skill',
-                                     json=example_skill).json['id']
 
-    response = app.test_client().get('/resume/skill')
-    assert response.json[item_id] == example_skill
+def test_add_skill():
+    response = app.test_client().post('/resume/skill', json={
+        "name": "Python",
+        "proficiency": "15%",
+        "logo": "example-logo.png"
+    })
+    print(response.json)
+    assert response.status_code == 201
+
+if __name__ == '__main__':
+    test_add_experience()
+    test_add_education()
+    test_add_skill()
```

Comment:
This part is not necessary because it's already automated with Pytest

Reply:
Okay alright! I will add a commit that removes this part 🚀